pkg/local_object_storage/shard/restore.go

```diff
@@ -0,0 +1,122 @@
+package shard
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"io"
+	"os"
+
+	"github.com/nspcc-dev/neofs-node/pkg/core/object"
+)
+
+// ErrInvalidMagic is returned when dump format is invalid.
+var ErrInvalidMagic = errors.New("invalid magic")
+
+// RestorePrm groups the parameters of Restore operation.
+type RestorePrm struct {
+	path         string
+	ignoreErrors bool
```

Comment:
should the `Evacuate` operation have the same `ignoreErrors`? i think evacuating "all I can" is even more important than restoring

Reply:
Fixed.